src/App.jsx

```diff
@@ -1,7 +1,24 @@
-export default function App() {
+import { AnimalListItem } from "./components/AnimalListItem";
+
+const App = () => {
+  const animals = [
```

Comment:
And it would look more optimized if you use these data as props like pass them to the component where you need them! Don't you think so ?